packages/@aws-cdk/aws-lambda-event-sources/lib/stream.ts

```diff
@@ -41,7 +41,7 @@ export interface StreamEventSourceProps {
    * * Minimum value of 60 seconds
    * * Maximum value of 7 days
    *
-   * @default Duration.days(7)
+   * @default - the record is available for processing until it expires
```

Comment:
Isn't this a bit cyclic? Don't we want to know what the actual lifetime is?

Reply:
This will depend on the AWS service's own default and any customization the user might have done to the original resource.
For example, in Kinesis, the default (and max) is 24h but can be configured to any value less than that.

How do you propose we document this?

Reply:
`@default - the expiration delay configured on the Kinesis stream` ?

Reply:
```suggestion
   * @default - the retention period configured on the stream
```